src/commands/limits/api/display.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as os from 'os';
+import { Messages, SfError } from '@salesforce/core';
+import { Flags, SfCommand, orgApiVersionFlagWithDeprecations, loglevel } from '@salesforce/sf-plugins-core';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-limits', 'display');
+
+type ApiLimit = {
+  name: string;
+  max: number;
+  remaining: number;
+};
+
+interface Result {
+  [key: string]: {
+    Max: number;
+    Remaining: number;
+  };
+}
+
+export type ApiLimits = ApiLimit[];
+
+export class LimitsApiDisplayCommand extends SfCommand<ApiLimits> {
+  public static aliases = ['force:limits:api:display', 'org:list:limits'];
+  public static readonly summary = messages.getMessage('description');
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
```

Comment:
prefer getMessages for examples (avoids the split, os import).  I learned that trick from Mike.